abstio/src/abst_data.rs

```diff
@@ -96,6 +101,22 @@ impl Manifest {
             || name == "west_seattle"
             || name == "udistrict"
     }
+
+    /// If an entry's path is system data, return the city.
+    pub fn path_to_city(path: &str) -> Option<CityName> {
+        let parts = path.split("/").collect::<Vec<_>>();
+        if parts[1] == "system" {
+            if parts[2] == "assets"
+                || parts[2] == "extra_fonts"
+                || parts[2] == "proposals"
+                || parts[2] == "study_areas"
+            {
+                return None;
+            }
```

Comment:
It's a little awkward that the country directories (`us`, `pl`, etc.)  are at the same level as all these fixed directories (`assets`, `extra_fonts`). 

It might be simpler for them to all live in `system/country` or somesuch, but might not be worth the pain of migrating everything, so take it or leave it.

Reply:
Yeah I regret organizing things this way. It may be a little easier to move all the extra directories into `data/assets`, or maybe even out of `data/` entirely. Probably `data/` should just be things not under version control and downloaded dynamically.